### PR TITLE
[quant][graphmode][fx] Add support for add

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43377 [quant][graphmode][fx] Add support for instance_norm
* #43376 [quant][graphmode][fx] Add support for layer_norm
* #43375 [quant][graphmode][fx] Add support for elu
* #43374 [quant][graphmode][fx] Add support for hardswish
* #43373 [quant][graphmode][fx] Add support for mul and mul relu
* #43372 [quant][grapphmode][fx][test][refactor] Refactor quantized add test
* #43335 [quant][graphmode][fx] Add support for batchnorm relu
* #43334 [quant][graphmode][fx] Add support for batchnorm
* #43333 [quant][graphmode][fx] Add support for cat
* #43332 [quant][graphmode][fx] Add support for add relu
* **#43331 [quant][graphmode][fx] Add support for add**
* #43287 [quant][graphmode][fx] Add support for conv module + relu

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243561](https://our.internmc.facebook.com/intern/diff/D23243561)

### CODE REVIEW

File: test/quantization/test_quantize_fx.py [hunk 1]
```diff
@@ -206,3 +207,109 @@ def forward(self, x):
...
+                self.conv1 = torch.nn.Conv2d(2, 2, 2).float()
+                self.conv2 = torch.nn.Conv2d(2, 2, 2).float()
+
+            def forward(self, x, y):
+                x = self.conv1(x)
+                y = self.conv2(y)
+                x += y
+                return x
+
+        # TODO: decide whether we want to quantize or not
```
Comment (raghuramank100) [inline test/quantization/test_quantize_fx.py:237]: Why is this case not quantized?

**Review by raghuramank100 — APPROVED** [review]
